REQUIRE

```diff
@@ -7,3 +7,4 @@ Reexport
 FixedSizeArrays
 Measures
 Showoff
+StatsBase
```

Comment:
0.14.0 - it is already in METADATA. I'll try to review quickly and push a new minor version release.